README.md

```diff
@@ -10,31 +10,74 @@ With the public release of `$vectorSearch`, we have needed to integrate into the
+```bash
+├── LICENSE                                     # License Agreeement
+├── README.md                                   # This Document
+├── langchain-python                            # Folder scoped for one Integration
+│   └── run.sh                                  # script that executes test
+├── semantic-kernel-csharp                      # Folder scoped for one Integration
+│   ├── database                                # Optional database definition
+│   │   └── nearestSearch.json                  # Populates $DATABASE.nearestSearch
+│   │   └── furthestSearch.json                 # Populates $DATABASE.furthestSearch
+│   ├── indexConfig.json                        # Creates Search Index on $DATABASE
+│   └── run.sh                                  # script that executes test
+├── semantic-kernel-python                      # Folder scoped for one Integration
+│   ├── database                                # Optional database definition
+│   │   └── nearestSearch.json                  # Populates $DATABASE.nearestSearch
+│   │   └── furthestSearch.json                 # Populates $DATABASE.furthestSearch
+│   ├── indexConfig.json                        # Creates Search Index on $DATABASE
+│   └── run.sh                                  # script that executes test
+```
+
+### Configuring a Atlas CLI for testing
+Each test subdirectory will automatically have its own local Atlas deployment. As a result, database and collection names will not conflict between different AI/ML integrations. To connect to your local Atlas using a connection string, call `$atlas` from the `run.sh` script within your subdirectory. This exposes the Atlas CLI binary. For example:
+
+```bash
+CONN_STRING=$($atlas deployments connect $DIR --connectWith connectionString)
+```
+Stores the local Atlas URI within the `CONN_STRING` var. It can then pass `CONN_STRING` as an environment variable to the test suite.
+
+#### Pre-populating the Local Atlas Deployment
+You can pre-populate a test's local Atlas deployment before running the `run.sh` script by providing JSON files in the optional `database` directory of the created subdirectory. The `.evergreen/scaffold_atlas.py` file will search for every JSON file within this database directory and upload the documents to the database provided by the `DATABASE` expansion provided in the buildvariant of the `.evergreen/config.yml` setup. The collection the script uploads to is based on the name of your JSON file:
+- `critical_search.json` with `DATABASE: app` will upload all of its contents to `app.critical_search` collection.
+
+To create a search index, provide the search index configuration in the `indexConfig.json` file. The evergreen script will then create the search index before the `run.sh` script is executed. Adding multiple search indexes in the setup stage is not supported, but more indexes can be included in the `run.sh` by using the referenced `$atlas` binary.
```

Comment:
Is adding multiple search indexes documented anywhere? Or is it covered in the CLI docs?

Reply:
It's not covered in the CLI docs. I imagine they're all single calls. I'll add 
```suggestion
To create a search index, provide the search index configuration in the `indexConfig.json` file. The evergreen script will then create the search index before the `run.sh` script is executed. Adding multiple search indexes in the setup stage is not supported, but more indexes can be included in the `run.sh` by using the referenced `$atlas` binary. See the ["Create an Atlas Search Index and Run a Query"](https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-fts/#create-an-atlas-search-index-and-run-a-query) documentation instructions on how to create a search index using Atlas CLI.
```